doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -75,6 +225,124 @@ The following UICR fields are supported:
    If no UICR values are programmed, |ISE| applies a set of default configurations.
    Applications that do not require custom settings can rely on these defaults without modifying the UICR.
 
+UICR image generation
+=====================
+
+The UICR image is an automatically created build artifact that generates the UICR content and peripheral configuration (PERIPHCONF) entries.
+
+What is the UICR image
+----------------------
+
+The UICR image consists of several components:
+
+* UICR configuration - The main UICR structure containing version information, security settings, memory protection, and boot configuration
+* PERIPHCONF entries - Peripheral configuration data extracted from the devicetree that defines peripheral access permissions and pin multiplexing
+
+The UICR image is generated as a HEX file.
```

Comment:
What?

We should have already stated that the "UICR image" is a zephyr image, in which case we can state that the uicr hex file is generated when building the image.

Do we strictly need this information if we have already described that its a zephyr image and what it contains?

Reply:
pushed a new version with a table.

Please have another look